src/engine.rs

```diff
@@ -294,25 +275,11 @@ pub fn run_treefmt(
     ret
 }
 
-fn print_summary(
-    traversed_files: usize,
-    matched_files: usize,
-    filtered_files: usize,
-    reformatted_files: usize,
-    start_time: std::time::Instant,
-) {
+fn print_summary(traversed_files: usize, reformatted_files: usize, start_time: std::time::Instant) {
     println!(
-        r#"
-traversed {} files
-matched {} files to formatters
-left with {} files after cache
-of whom {} files were re-formatted
-all of this in {:.0?}
-        "#,
-        traversed_files,
-        matched_files,
-        filtered_files,
+        "\n{} file(s) changed (processed {} file(s) in {:.0?})\n",
         reformatted_files,
+        traversed_files,
```

Comment:
Most of the time is usually dominated by the formatters and is in function of the number of files that are being processed by the formatters. We were arguing on slack to display the filtered_files here because of that.